.github/workflows/typescript.yml

```diff
@@ -63,6 +74,73 @@ jobs:
       - name: Tests
         run: yarn test
 
+  typescript-build-and-deploy-testnet:
+    needs: [typescript-detect-changes, typescript-build-and-test]
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./typescript
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+          cache: "yarn"
+          cache-dependency-path: typescript/yarn.lock
+          registry-url: "https://registry.npmjs.org"
+
+      # We need this step because the `@keep-network/tbtc-v2` which we update in
+      # next step has an indirect dependency to `@summa-tx/relay-sol@2.0.2`
+      # package, which downloads one of its sub-dependencies via unathenticated
+      # `git://` protocol. That protocol is no longer supported. Thanks to this
+      # step `https://` is used instead of `git://`.
+      - name: Configure git to don't use unauthenticated protocol
+        run: git config --global url."https://".insteadOf git://
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Get upstream packages versions
+        uses: keep-network/ci/actions/upstream-builds-query@v2
+        id: upstream-builds-query
+        with:
+          upstream-builds: ${{ github.event.inputs.upstream_builds }}
+          query: tbtc-v2-contracts-version = github.com/keep-network/tbtc-v2#version
```

Comment:
It also needs `@keep-network/ecdsa` contracts dependency.

Reply:
Right, it looks that I created a branch from the code that did not have the `ecdsa` dependency. I updated the code and added the `ecdsa` upgrade in [a7bb07b](https://github.com/keep-network/tbtc-v2/pull/522/commits/a7bb07bf8708535b6505077de91c431aed31a083).